.travis.yml

```diff
@@ -11,15 +11,15 @@ env:
   global:
     - GO111MODULE=on
     - PATH=bin:$PATH
-    - DOCKER_USERNAME=pix4d
+    - DOCKER_USERNAME=mmoltenipix4d
+    - DOCKER_ORG=pix4d
```

Comment:
Why both? Because tokens are given only to users, not organizations.
So we use the user to docker login and the org to tag the image :-/